skyplane/cli/cli_impl/cp_replicate.py

```diff
@@ -127,22 +127,25 @@ def generate_full_transferobjlist(
     dest_bucket: str,
     dest_prefix: str,
     recursive: bool = False,
+    requester_pays: str = None
 ) -> List[Tuple[ObjectStoreObject, ObjectStoreObject]]:
     """Query source region and destination region buckets and return list of objects to transfer."""
     source_iface = ObjectStoreInterface.create(source_region, source_bucket)
     dest_iface = ObjectStoreInterface.create(dest_region, dest_bucket)
 
+    if requester_pays == 'requester' and isinstance(source_iface, S3Interface):
+        source_iface.requester_pays = True
+
     # ensure buckets exist
     if not source_iface.bucket_exists():
         raise exceptions.MissingBucketException(f"Source bucket {source_bucket} does not exist")
     if not dest_iface.bucket_exists():
         raise exceptions.MissingBucketException(f"Destination bucket {dest_bucket} does not exist")
-
     source_objs, dest_objs = [], []
 
     # query all source region objects
     logger.fs.debug(f"Querying objects in {source_bucket}")
-    with console.status(f"Querying objects in {source_bucket}") as status:
+    with console.status(f"Querying objects in {source_bucket}\n") as status:
```

Comment:
Print the new line with the logger statement `logger.error("Specified object does not exist.")`